microcosm_elasticsearch/tests/test_store.py

```diff
@@ -313,3 +325,24 @@ def test_bulk_with_report(self):
             has_key('delete'),
         ))
         assert_that(result[1][0]['delete'], has_entry('result', 'not_found'))
+
+    def test_overloaded_store(self):
```

Comment:
I add a similar case for the `search` operation, as the code paths for searching can differ from other operations.